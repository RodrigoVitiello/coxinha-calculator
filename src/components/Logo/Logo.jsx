import React from 'react';
import logo from './assets/logo.png';
import './Logo.scss'

const Logo = () => {
  return <img src={logo} className="logo" alt="rotating-coxinhas" />
}

export default Logo;
