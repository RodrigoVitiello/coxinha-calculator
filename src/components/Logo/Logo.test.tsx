import React from 'react';
import { render } from '@testing-library/react';
import Logo from './Logo';

it('renders coxinhas logo component', () => {
  const { container } = render(<Logo />);

  expect(container).toMatchSnapshot();
});
