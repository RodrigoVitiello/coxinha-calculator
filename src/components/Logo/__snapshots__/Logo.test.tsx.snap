// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coxinhas logo component 1`] = `
<div>
  <img
    alt="rotating-coxinhas"
    class="logo"
    src="logo.png"
  />
</div>
`;
